modules/steam/network.nix

```
@@ -0,0 +1,24 @@
+{lib, config, pkgs, ...}: let
+  cfg = config.jovian.steam;
+in with lib; {
+  options.jovian.steam.network = {
+    enable = mkOption {
+      description = mdDoc "Open required ports for various Steam features.";
+      type = types.bool;
+      default = cfg.enable;
```

```suggestion
      default = cfg.enable;
      defaultText = lib.literalExpression "config.jovian.steam.enable";
```

The docs would otherwise state the default is a bool without hinting at the way it gets set.